include/seqan3/core/range/detail/inherited_iterator_base.hpp

```diff
@@ -109,18 +109,21 @@ class inherited_iterator_base : public std::conditional_t<std::is_pointer_v<base
         : member{std::move(it)}
     {}
     //!\}
+    //
 
-    //!\brief Get a copy of the base.
-    constexpr base_t base() const &
-    //!\cond
-        requires std::copyable<base_t>
-    //!\endcond
+    constexpr base_t const & base() const & noexcept
```

Comment:
If I'm correct, the Doc tests failed because they two function are not documented. Just a suggestion:
```suggestion
    //!\brief Get a const reference to the base.
    constexpr base_t const & base() const & noexcept
```